### ISSUE TITLE
_episodes_rmd/05-Gradient-Boosting.Rmd: Minor text edit

### ISSUE THREAD

**Issue by gmcdonald-sfg** [djhunter/r-ml-tabular-data#22]
In the “Learning Rate” section, suggest introducing the concept of hyperparameters. Learning rate is a type of hyperparameter specific to xgboost.

**Comment by djhunter** [comment]
Learning rate is not specific to xgboost.
